### PR TITLE
Refine TF NLP models with INC User NewAPI

### PR DESCRIPTION
**Pull request by ChendaLi-Intel**
## Type of Change
Examples
API not changed

## Description
JIRA ticket: [https://jira.devtools.intel.com/browse/ILITV-2454](https://jira.devtools.intel.com/browse/ILITV-2454)

## Expected Behavior & Potential Risk
Pre-CI Test pass
Extension Test pass

## How has this PR been tested?
Pre-CI Test
Extension Test

### CODE REVIEW

**Comment by ChendaLi-Intel** [comment]
Please launch the Extension Test after the PR below is merged into master:
https://github.com/intel/neural-compressor/pull/186

**Comment by ChendaLi-Intel** [comment]
Please pull master after the PR below is merged into master and launch the Extension Test.
https://github.com/intel/neural-compressor/pull/186

**Comment by ChendaLi-Intel** [comment]
distilbert_base:
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3566/

bert_base_mrpc:
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3567/

**Review by ftian1 — CHANGES_REQUESTED** [review]

File: examples/tensorflow/nlp/distilbert_base/quantization/ptq/README.md [hunk 1]
```diff
@@ -138,45 +136,50 @@ class Dataloader(object):
...
+set_random_seed(9527)
+tuning_criterion = TuningCriterion(
+    strategy="basic",
+    timeout=0,
+    max_trials=100,
+    objective="performance")
+
+tolerable_loss = AccuracyLoss(loss=0.02)
+
+accuracy_criterion = AccuracyCriterion(
```
[the remark this answers is not in the document]
Comment (ftian1) [reply examples/tensorflow/nlp/distilbert_base/quantization/ptq/README.md]: sorry, I meant it should be a scalar for tolerable_loss variable.

[the remark this answers is not in the document]
Comment (zehao-intel) [reply examples/tensorflow/nlp/distilbert_base/quantization/ptq/README.md]: Got it. This has been modified.

File: examples/tensorflow/nlp/bert_base_mrpc/quantization/ptq/README.md [hunk 2]
```diff
@@ -62,129 +63,76 @@ bash prepare_model.sh --dataset_location=./data --output_dir=./model
...
-      quantizer.eval_dataloader = common.DataLoader(dataset, collate_fn=collate_fn)
-      quantizer.metric = Accuracy()
-      q_model = quantizer.fit()
-      q_model.save(FLAGS.output_model)
+    from neural_compressor import quantization
+    from neural_compressor.config import PostTrainingQuantConfig
+    config = PostTrainingQuantConfig(
+        inputs=["input_file", "batch_size"],
+        outputs=["loss/Softmax:0", "IteratorGetNext:3"],
+        calibration_sampling_size=[500],)
```
Comment (ftian1) [inline examples/tensorflow/nlp/bert_base_mrpc/quantization/ptq/README.md:116]: if sampling_size is single number, the underneath logic should be able to accept scaler but not only list. @PenghuiCheng

File: examples/tensorflow/nlp/transformer_lt_mlperf/quantization/ptq/README.md [hunk 3]
```diff
@@ -131,51 +130,52 @@ class Dataset(object):
...
-q_model = quantizer.fit()
-q_model.save(FLAGS.output_model)
+  from neural_compressor import quantization
+  from neural_compressor.experimental import common
+  from neural_compressor.config import PostTrainingQuantConfig
+  dataset = Dataset(FLAGS.input_file, FLAGS.vocab_file)
+  calib_dataloader = common.DataLoader(dataset,
+                                        collate_fn = collate_fn,
+                                        batch_size = FLAGS.batch_size)
+  input_model = common.Model(graph)
```
Comment (ftian1) [inline examples/tensorflow/nlp/transformer_lt_mlperf/quantization/ptq/README.md:161]: is it a pb? if yes, you can directly pass graph into fit() but not wrap with common.Model

Comment (zehao-intel) [reply examples/tensorflow/nlp/transformer_lt_mlperf/quantization/ptq/README.md:161]: It's a graph_def. The common.Model has been removed. Thanks for comments.

**Comment by zehao-intel** [comment]
Rebuild test: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3734/

**Comment by chensuyue** [comment]
> Extension: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3750/

Please check perf gap of 'transformer_lt_mlperf'.

**Comment by zehao-intel** [comment]
Rebuild: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3766/

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3770/)

**Comment by zehao-intel** [comment]
Distillbert: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3774/

**Comment by chensuyue** [comment]
Retest after copy data into workspace: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3848/

**Comment by chensuyue** [comment]
> Retest after copy data into workspace: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3848/

retest distillbert with spr-base: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3859/

**Comment by zehao-intel** [comment]
DistillBert Base: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3863/
Bert Base MRPC: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3862/

**Comment by chensuyue** [comment]
Final issue, please update this page: https://github.com/intel/neural-compressor/blob/chenda/inc_newapi/examples/README.md

**Pull request merged**